app/services/foreman/plugin.rb

```diff
@@ -243,7 +243,10 @@ def pending_migrations
       pending_migrations = ActiveRecord::Migrator.new(:up, migration_paths).
         pending_migrations
 
-      pending_migrations.size > 0
+      return false if pending_migrations.size.zero?
+      Rails.logger.debug(
+        "Pending migrations: #{pending_migrations.map(&:name).join(', ')}")
```

Comment:
Does the whole list need to be printed here? Can it be the first 5 entries, optionally with a count included?

It doesn't sound like the contents are very important, which might prevent this being a huge message that increases over time.